eng/ci/official-build.yml

```diff
@@ -37,6 +37,11 @@ extends:
       name: 1es-pool-azfunc
       image: 1es-windows-2022
       os: windows
+    sdl:
+      codeql:
+         compiled:
+           enabled: true
+         runSourceLanguagesInSourceAnalysis: true
     stages:     
```

Comment:
do you need to add language and cadence as well as the variables?

Reply:
No need to add the language.  Will leave default cadence to 72.